app/models/experiment.rb

```diff
@@ -499,7 +499,12 @@ def move_modules(to_move)
         raise ActiveRecord::ActiveRecordError
       end
 
-      my_module.save!
+      my_module.save
+    end
+
+    # Generate workflow image for the experiment in which we moved the task
+    to_move.each_value do |id|
+      experiment = Experiment.find_by_id(id)
       experiment.delay.generate_workflow_img
```

Comment:
Please no h8 m8, but it's possible that experiment id doesn't exist, so it needs check for `if present`